libs/file-storage/src/lib/file-storage.service.ts

```diff
@@ -37,31 +47,25 @@ export class FileStorageService {
     const fileId = uuid()
     const key = `${fileId}_${kebabCase(fName)}${fExt}`
 
-    const params = {
+    const { url, fields } = await createPresignedPost(this.s3Client, {
       Bucket: this.config.uploadBucket,
-      Expires: PRESIGNED_POST_EXPIRES,
-      Fields: {
-        key,
-      },
-      conditions: [['content-length-range', 0, 10000000]], // Max 10MB
-    }
-
-    return new Promise((resolve, reject) => {
-      this.s3.createPresignedPost(params, (err, data) => {
-        if (err) {
-          reject(err)
-        } else {
-          resolve(data)
-        }
-      })
+      Key: key,
+      Conditions: [
+        ['content-length-range', 0, 10000000], // Max 10MB
+      ],
+      Expires: PRESIGNED_POST_EXPIRES / 1000, // createPresignedPost expects seconds
```

Comment:
~Can just update the constant further up in this file.

Reply:
True, I changed the constant value and removed the division by 1000.